packages/botonic-plugin-contentful/src/cms/contents.ts

```diff
@@ -171,6 +172,7 @@ export class CommonFields implements Stringable {
       partitions?: string[]
       dateRange?: DateRangeContent
       followUp?: FollowUp
+      customFields?: Record<string, unknown>
```

Comment:
add a comment:
"customFields are any fields found in CMS but not expected in the corresponding model".
@vanbasten17 @guinii @asastre should we call it customFields or extraFields?

Reply:
I'm ok with `customFields`.